### PR TITLE
Adding inject-shellcode-using-mapping-injection.yml

### PR DESCRIPTION
**Pull request by jtothej**
New rule that detects process injection using the Mapping Injection technique.

### CODE REVIEW

File: host-interaction/process/inject/inject-shellcode-using-mapping-injection.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+rule:
+  meta:
+    name: inject shellcode using Mapping Injection
```
Comment (williballenthin) [inline host-interaction/process/inject/inject-shellcode-using-mapping-injection.yml:3]:
```suggestion
    name: inject shellcode using a file mapping object
```

imho, "Mapping Injection" isn't a a well known term and the technique seems to be a fairly straightforward application of file mapping objects, so let's just describe it as that

**Review by williballenthin — APPROVED** [review]
logic looks good, though some overlap with here:
https://github.com/mandiant/capa-rules/blob/26d3a7937f826169588d356f5af1a7bc1211e365/host-interaction/process/inject/inject-apc.yml

this rule is more succinct, though.

one recommendation around the rule name. let me know what you think.

File: host-interaction/process/inject/inject-shellcode-using-mapping-injection.yml (hunk 1 above)
Comment (jtothej) [reply host-interaction/process/inject/inject-shellcode-using-mapping-injection.yml]: Sure, that makes perfect sense. I submitted the renamed rule.

**Comment by williballenthin** [comment]
thank you!

**Pull request merged**